mgmt/rest/v2/plugin.go

```diff
@@ -183,40 +211,44 @@ func (s *apiV2) loadPlugin(w http.ResponseWriter, r *http.Request, _ httprouter.
 				}
 			}
 
-			// A little sanity checking for files being passed into the API server.
-			// First file passed in should be the plugin. If the first file is a signature
-			// file, an error is returned. The signature file should be the second
-			// file passed to the API server. If the second file does not have the ".asc"
-			// extension, an error is returned.
-			// If we loop around more than twice before receiving io.EOF, then
-			// an error is returned.
-
-			switch {
-			case i == 0:
-				if filepath.Ext(p.FileName()) == ".asc" {
-					e := errors.New("Error: first file passed to load plugin api can not be signature file")
-					Write(400, FromError(e), w)
+			switch p.FormName() {
+			case "ca_certs":
+				fn, err := createTempFile(b, TLSCACertsPrefix)
```

Comment:
@Iza, I would be happy to change the implementation if we can assume those certs and the key files are always accessible. Note that I implemented `snaptel` in V2 using "https" for this. We can remove https as well. Please let me know you guys take on this.